logback/src/main/java/com/linecorp/armeria/common/logback/RequestContextExportingAppender.java

```diff
@@ -144,7 +147,8 @@ public void setExport(String mdcKey) {
      */
     public void setExports(String mdcKeys) {
         requireNonNull(mdcKeys, "mdcKeys");
-        builder.addKeyPattern(mdcKeys);
+        checkArgument(!mdcKeys.isEmpty(), "mdcKeys must not be empty");
+        Arrays.stream(mdcKeys.split(",")).map(String::trim).forEach(builder::addKeyPattern);
```

Comment:
I think we could optimize this when `mdcKeys.split(",").length == 1`.

Reply:
`setExports` looks to be called only once.
Do we need an optimization?

Reply:
That could be called multiple times according to the `logback.xml`.
By the way, it seems overkill. Let's leave it as it is. 😀

Reply:
Let's use the splitter from Guava so that we don't have to trim again. 😉 
```java
private static final Splitter KEY_SPLITTER = Splitter.on(',').trimResults();
...
KEY_SPLITTER.split(mdcKeys).forEach(builder::addKeyPattern);
```

Reply:
Thanks 🙏 
I use it https://github.com/line/armeria/pull/2781/commits/ab8e780f33aeb5007ff8072232282fb4b0fdea5f